src/components/SideNavbar.tsx

```diff
@@ -122,28 +119,28 @@ export default function SideNavbar(props: navbarProps) {
 let defaultNavItems: NavItems[] = [
```

Comment:
I think these should be passed into the Navbar as children elements, that way the whole thing is more composable. Additionally these items can be moved to a more intuitive location where the Sidebar is used and not hardcoded in a list.

Also, I'm not sure how much work it would take, but making the logout button not be hardcoded might be a good idea. There might be a way to define whether a NavItem is in the upper or lower portion of the Sidebar which will again make the Sidebar more modular and easier to use.